test/e2e/util.go

```diff
@@ -230,3 +233,32 @@ func CreateMachineSet(ctx context.Context, cfg *rest.Config, mc *machinesetclien
 
 	return mc.MachineSets(MachineAPINamespace).Create(ctx, machineset, metav1.CreateOptions{})
 }
+
+func getMachines(ctx context.Context, dc *dynamic.DynamicClient) (*machinev1beta1.MachineList, error) {
+	machineClient := dc.Resource(schema.GroupVersionResource{Group: "machine.openshift.io", Resource: "machines", Version: "v1beta1"})
+	machineObjxList, err := machineClient.List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("Unable to check for machines: %v", err)
+	}
+
+	machineObjxListBytes, err := machineObjxList.MarshalJSON()
+	if err != nil {
+		return nil, fmt.Errorf("unable to marshal JSON: %v", err)
+	}
+	var machineList machinev1beta1.MachineList
+
+	if err = json.Unmarshal(machineObjxListBytes, &machineList); err != nil {
+		return nil, fmt.Errorf("unable to marshal JSON: %v", err)
+	}
+	return &machineList, nil
+}
+
+func isIpInCidrRange(ip string, cidr string) (bool, error) {
```

Comment:
I see this is duplicated in this file and in the vsphere/util.go file - I think I'm missing something, what's the difference?

Reply:
hmmm ... didnt mean to duplicate that.  i'll make vsphere/util.go public.

Reply:
yeah, it was an errant duplication, fixing it now